src/locales/cy/translation.json

```diff
@@ -291,6 +291,10 @@
         "header": "Rhowch eich cyfrinair",
         "paragraph": "Mae angen i ni wneud yn siŵr mai chi yw chi cyn i chi allu rheoli sut rydych yn cael codau diogelwch."
       },
+      "globalLogout": {
+        "header": "Rhowch eich cyfrinair",
+        "paragraph": "Mae angen i ni wneud yn siŵr mai chi yw chi cyn i chi allu rheoli sut rydych yn cael codau diogelwch."
```

Comment:
Security codes? Does this content need to be updated something like "Mae angen i ni sicrhau mai chi ydyw cyn y gallwch chi allgofnodi o bob dyfais.?